sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_loader_interface.cpp

```diff
@@ -86,7 +86,6 @@ UR_DLLEXPORT ur_result_t UR_APICALL urGetEnqueueProcAddrTable(
   pDdiTable->pfnMemUnmap = nullptr;
   pDdiTable->pfnUSMMemcpy = nullptr;
   pDdiTable->pfnUSMPrefetch = nullptr;
-  pDdiTable->pfnUSMMemAdvise = nullptr;
```

Comment:
why was this removed? this is still on UR

https://github.com/oneapi-src/unified-runtime/blob/416b86a2c3de9e7104cf3ba5d11f064eff9c10fe/include/ur_api.h#L5219

Reply:
It was renamed,  this variable is not in table anymore

Reply:
@bmyates  could you add the new name, instead of removing it?

https://github.com/oneapi-src/unified-runtime/blob/416b86a2c3de9e7104cf3ba5d11f064eff9c10fe/include/ur_ddi.h#L1042

```cpp
    ur_pfnEnqueueUSMPrefetch_t pfnUSMPrefetch;
    ur_pfnEnqueueUSMAdvise_t pfnUSMAdvise;
    ur_pfnEnqueueUSMFill2D_t pfnUSMFill2D;
```

Reply:
added